### PR TITLE
PB-5850: Add code to create global rules for all cases

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Add code to create global rules for all cases

Dashboard url:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/nonpx/job/ibm/job/s3/job/roks/job/ibm-roks-s3-direct-kdmp/11/console
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/nonpx/job/ibm/job/s3/job/roks/job/ibm-roks-s3-direct-kdmp/9/console (Previous run before ownership update fix)

**Which issue(s) this PR fixes** (optional)
Closes #PB-5850

**Special notes for your reviewer**:

### CODE REVIEW

**Review by sn-px — APPROVED** [review]
LGTM

File: tests/backup_helper.go [hunk 1]
```diff
@@ -145,6 +145,11 @@ var (
 	GlobalAWSLockedBucketName   string
 	GlobalAzureLockedBucketName string
 	GlobalGCPLockedBucketName   string
+	GlobalPreRuleName           string
+	GlobalPostRuleName          string
+	GlobalPreRuleUid            string
+	GlobalPostRuleUid           string
+	FlagUseGlobalRule           bool
```
Comment (mkoppal-px) [inline tests/backup_helper.go:152]: nit: Can we rename this to `UseGlobalRule` or `GlobalRuleFlag` ?

**Comment by mkoppal-px** [comment]
@kshithijiyer-px The code looks good. Can we just do a sample run by setting it to false and check? Even 1 test case is enough.

**Review by mkoppal-px — APPROVED** [review]
LGTM except for one minor comment

File: tests/backup_helper.go (hunk 1 above)
Comment (kshithijiyer-px) [reply tests/backup_helper.go:152]: Done

**Comment by kshithijiyer-px** [comment]
> @kshithijiyer-px The code looks good. Can we just do a sample run by setting it to false and check? Even 1 test case is enough.

I accidentally did it on one of the BYOC Runs you can refer this one it'll work. 
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/Cloud_BYOC/144/

**Pull request merged**